hack/generate/csv.sh

```diff
@@ -91,7 +118,16 @@ cp "$template" "$target"
 
 for name in "${images[@]}"; do
   echo "Image: ${name} -> ${images_addresses[$name]}"
-  add_image "$target" "$name" "${images_addresses[$name]}"
```

Comment:
This split might be unnecessary if we handle the prefixes differently as proposed. Sure, the images are not needed on the upstream operator but I don't think they'd hurt either.

Reply:
Still need to track if an image is a Kafka image since they need to have a different prefix.
If that's ok, let's leave this as is. In the future, we can actually add scopes to eventing and serving (so that we don't pass eventing overrides to serving CR) and then we can find a unified solution.